### DISCUSSION TITLE
Does the ImageSharp support converting ICCProfile into the srgb?

### DISCUSSION THREAD

**Discussion by vipulpatel2008** [SixLabors/ImageSharp#2491] [Q&A] [answered]
I have assign the IccProfile to the Image.Metadata.IccProfile object. It work well but the browser doesn't recognize the attached IccProfile. 
My planning is to convert IccProfile into the srgb.
Please let me know if it is achievable or any other idea that I can help to remove browser rendering issues of the image.

**Comment by JimBobSquarePants** [comment, marked as answer]
Not yet, see https://github.com/SixLabors/ImageSharp/pull/1567
